src/tasks/tasks.controller.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { TaskController } from './tasks.controller';
+import { TaskService } from './tasks.service';
+import { TaskStatus } from '../tasks/task-status.enum';
+
+describe('TaskController', () => {
+  let controller: TaskController;
+  let fakeTaskService: Partial<TaskService>;
+
+  beforeEach(async () => {
+    fakeTaskService = {
+      createTask: () => {
+        return Promise.resolve({
+          id: '3',
```

Comment:
Even if these are fake object, I think it can be good to be consistency on the format of uuid of 16 characters so we can not confuse anyone who can review these codes thinking we are using the other type of id.